### PR TITLE
Add GleecBTC Coin

### PR DESCRIPTION
**Pull request by shamimiceewu025**
Need Add GleecBTC Coin

### CODE REVIEW

**Comment by prusnak** [comment]
Which wallet is going to integrate Trezor?

File: common/defs/bitcoin/gleacebtc.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "coin_name": "GleecBTC",
+  "coin_shortcut": "Gleec",
+  "coin_label": "GleecBTC Coin",
+  "website": "https://gleecoin.com",
+  "github": "https://github.com/Gleecs/GleecBTC-FullNode-Win-Mac-Linux",
+  "maintainer": "Jochen Hoenicke <hoenicke@gmail.com>",
```
Comment (prusnak) [inline common/defs/bitcoin/gleacebtc.json]: This is wrong - put your own email here.

Comment (shamimiceewu025) [reply common/defs/bitcoin/gleacebtc.json]: Update Done. Also, mistake was File name

**Comment by prusnak** [comment]
Which GleecBTC wallet is going to integrate Trezor?

**Comment by shamimiceewu025** [comment]
Want To Integrated  GleecBTC coin into trezor T hardware Wallet

**Comment by prusnak** [comment]
I am asking which GleecBTC will integrate Trezor.

**Comment by prusnak** [comment]
Currently we are not accepting coins which are not in TOP 30 of CoinMarketCap. Please reopen the PR when your coin is there.

**Pull request closed without merging**